src/osd/OSD.cc

```diff
@@ -6833,11 +6833,16 @@ void OSD::sched_scrub()
         break;
       }
 
+      if ((scrub.deadline >= now) && !(time_permit && load_is_low)) {
```

Comment:
even if it's not allowed to schedule this scrub job at this moment, we can still try with the next one. so i think we should "continue" instead of "break" here.